app/src/main/java/com/m2049r/xmrwallet/layout/NodeInfoAdapter.java

```diff
@@ -52,6 +52,8 @@ public interface OnInteractionListener {
 
     private final FragmentActivity activity;
 
+    private Context context;
```

Comment:
there must be a better way of setting the context than from all the setNodes methods

Reply:
How?